test/distributed/fsdp/test_fsdp_ignored_modules.py

```diff
@@ -225,6 +227,23 @@ def test_diff_ignored_modules_across_ranks(
         optim = torch.optim.Adam(wrapped_model.parameters(), lr=1e-3)
         self._train_model(wrapped_model, optim, 3)
 
+    @skip_if_lt_x_gpu(2)
+    def test_ignored_modules_not_under_wrapped_root(self):
+        model = Model().cuda()
+        ignored_modules = list(model.layer1.children())[1:]
+        model.layer1 = FSDP(
+            model.layer1,
+            sharding_strategy=ShardingStrategy.SHARD_GRAD_OP,
```

Comment:
I am wondering if you think there is any benefit to leave a comment somewhere in this test to say that sharding strategy should not matter or at least some comment explaining the choice of sharding strategy (e.g. `model.layer1` vs. `model.layer3`).

Reply:
I am also wondering if you think it would be worth adding a comment to highlight how the ignored modules for the second application of FSDP to `model.layer3` is where the new code path is being tested, i.e. that there are ignored modules passed to L241 that are not in `model.layer3`'s tree.

Reply:
Hey @awgu, thanks for the quick stamp. Let me add those comments.